src/documents/tutorial-userInterfaceOptions/UserInterfaceOptions.md

```diff
@@ -37,20 +37,21 @@ The rest of this tutorial will explain each of these steps in detail.
 
 ## Download and install the Infusion library ##
 
-1. Get the current source code from github as a [ZIP file]( https://github.com/fluid-project/infusion/archive/master.zip)
-2. Unpack the zip file you just downloaded and cd into the "infusion-master" folder that results.
-3. If necessary install [Node.js](http://nodejs.org/download/) and [Grunt](http://gruntjs.com/) `npm install -g grunt-cli`.
-4. Make your own custom build by running the `grunt` command in the Terminal. See the [README.md](https://github.com/fluid-project/infusion/blob/master/README.md#how-do-i-create-an-infusion-package) file for instructions on how to make a custom build of Infusion.
-5. The grunt command will create a zip file in the products folder. Unzip that file and move the resulting `infusion` folder somewhere convenient for your development purposes, likely in a `lib` folder in your site hierarchy.
-6. This `infusion` will include a single file containing all of the JavaScript you need: `infusion-custom.js`. You will link to this file in the headers of your HTML files.
+1. Download the current source code from Github as a [ZIP file]( https://github.com/fluid-project/infusion/archive/master.zip).
+2. Unzip the downloaded ZIP file and change your directory into the `infusion-master` folder that results.
+3. Install [Node.js](http://nodejs.org/download/), if it hasn't been installed.
+4. Install [Grunt](http://gruntjs.com/getting-started) globally, if it hasn't been installed. For Windows, you can run `npm install -g grunt-cli` as Administrator. For Mac OSX, you can run `sudo npm install -g grunt-cli`.
+5. Create your build by running the `grunt` command in the Terminal. If you would like to customize your build, you can create a custom build by running the `grunt custom` command in the Terminal. See the [README.md](https://github.com/fluid-project/infusion/blob/master/README.md#how-do-i-create-an-infusion-package) file for instructions on how to make a custom build of Infusion.
+6. A ZIP file will be created in the `infusion-master/products` folder. Change directory to your preferred location for your development purposes and unzip the ZIP file in that directory, resulting in the folder: `infusion`. Once the resulting ZIP file is relocated, the other files in the `infusion-master` folder can be safely deleted. 
```

Comment:
Missing "the" "Change directory" -> "Change the directory".

Also the wording of this point is somewhat confusing. I think it could be clarified/simplified.